server/routes/termdb.topVariablyExpressedGenes.ts

```diff
@@ -110,13 +110,22 @@ async function computeGenes4nativeDs(
 	 * This is based on the recommendation of this article:
 	 * https://www.frontiersin.org/articles/10.3389/fgene.2021.632620/full.
 	 * This article recommends using interquartile region over variance.*/
+
+	let filter_extreme_values = true
+	if (q.filter_extreme_values == 1) {
+		filter_extreme_values = true
+	} else if (q.filter_extreme_values == 0) {
+		filter_extreme_values = false
+	}
+
 	const input_json = {
 		input_file: matrixFile,
 		samples: samples.join(','),
-		filter_extreme_values: true,
-		num_genes: q.maxGenes,
-		param: 'var'
+		filter_extreme_values: filter_extreme_values,
```

Comment:
simplify as filter_ex_v: q.filter_ex_v ? true : false

Reply:
The two statements are not equivalent. if (q.filter_extreme_values) is not defined (which is an optional variable) then the statement will be false. But filter_extreme_values should be "true" by default (in my opinion).

Reply:
then do `filter_extreme_values: q.filter_extreme_values == 1`